src/call.cpp

```diff
@@ -123,201 +123,50 @@ unsigned int call::wake()
 }
 
 #ifdef PCAPPLAY
-static char* find_sdp_eol(char* line)
+static std::string parse_sdp_msg(std::string const &pattern, std::string const &msg)
```

Comment:
A bit odd naming perhaps? Should it be called `find_in_sdp` or `get_sdp_line` (or similar) instead?